### DISCUSSION TITLE
Can I change the selection background color in the diff pane ?

### DISCUSSION THREAD

**Discussion by nicos68** [jesseduffield/lazygit#3239] [Q&A] [answered]
The line selection background color of the diff pane blends poorly with my terminal themes, be it the dark :
![image](https://github.com/jesseduffield/lazygit/assets/1062625/61a874a1-0628-4da6-9146-9d6d11fabe01)
or the light :
![image](https://github.com/jesseduffield/lazygit/assets/1062625/4dfacedc-b07f-46ce-bc25-31cf36964081)
For example `gui.theme.selectedLineBgColor` helped me a lot to improve the colors when I set it to `black`:
![image](https://github.com/jesseduffield/lazygit/assets/1062625/6df043b0-7879-4d61-a16f-8be94f602c27)
![image](https://github.com/jesseduffield/lazygit/assets/1062625/d9193577-a6aa-4391-9173-55e3634ccf2a)
Is there any similar option to improve that ?

**Comment by stefanhaller** [comment, marked as answer]
In the last release you can set `gui.theme.selectedRangeBgColor` for that. On master this has been removed though, `selectedLineBgColor` also applies to the diff pane there.

**Reply by nicos68** [reply]
Thanks, I used `selectedRangeBgColor` and now it looks perfect. I think my problems come from the fact that I run Fedora, and I don't have any reliable way to install and keep lazygit up to date. The copr is stuck at version 0.38.

So I had to delete and reinstall it using go. What's weird now is that if I use `selectedRangeBgColor` it works even though it's not listed in the `schema.json`, as my editor notices :
![image](https://github.com/jesseduffield/lazygit/assets/1062625/3b80d41f-f373-4ec2-9fc0-d3a94fe2cbfe)
But if I remove this property and only use `selectedLineBgColor`, it has no impact in the diff pane.

**Reply by stefanhaller** [reply]
How did you install it? If you use`go install github.com/jesseduffield/lazygit@latest`, you'll get the last tag, which is the same as the latest release. You'll have to say `@master` if you want master.

The schema that your editor uses is always the one from master though, as it has no way of knowing what version you are using. It's confusing, but we can't do much about that.

**Reply by nicos68** [reply]
Oh okay I never worked with go so I thought it would download the latest version from the default branch on github. It makes more sense now thanks.
